Google/Makefile

```diff
@@ -43,10 +43,18 @@ generate_credentials:
 	./fetch_config.rb -s azure-key-vault-secret:${KEY_VAULT}/${ASSET_SECRETS}  -f shell-env-var -d command -- jinja2 creds.json.tmpl > creds.json
 	clasp login --creds creds.json
 
+storage_parameters:
```

Comment:
`set-azure-account` should be added as a dependent target in all the targets where azure is used

Reply:
Added where it was missing